string action
---
string[] action_seq
float32[] action_prob
bool success